Resources/Prototypes/Entities/Objects/Weapons/Bombs/plastic.yml

```
@@ -54,6 +54,7 @@
     quickEquip: false
     equipDelay: 3
     unequipDelay: 6
+  - type: CatWearable
```

Add `# DeltaV` comment.

```suggestion
  - type: CatWearable # DeltaV
```